### PR TITLE
[type dropdown] fix for #3274

### PR DESCRIPTION
**Pull request by BurningTreeC**
this is a possible fix for the type dropdown where links don't show when "show missing links" is disabled

### CODE REVIEW

**Comment by Jermolene** [comment]
Thanks for tackling this @BurningTreeC. I've left a comment over at https://github.com/Jermolene/TiddlyWiki5/issues/3274#issuecomment-388829333

File: core/modules/widgets/link.js [hunk 1]
```diff
@@ -39,7 +39,7 @@ LinkWidget.prototype.render = function(parent,nextSibling) {
 		useWikiLinks = wikiLinksMacro ? (wikiLinksMacro.trim() !== "no") : true,
 		missingLinksEnabled = !(this.hideMissingLinks && this.isMissing && !this.isShadow);
 	// Render the link if required
-	if(useWikiLinks && missingLinksEnabled) {
+	if(useWikiLinks && missingLinksEnabled || this.showLinks) {
```
Comment (pmario) [inline core/modules/widgets/link.js]: I'm not so happy with the name. ... For me it is more like (lacking a better name) "forceLinks" ...

Comment (pmario) [reply core/modules/widgets/link.js]: or may be "enforceLinks"

Comment (BurningTreeC) [reply core/modules/widgets/link.js]: no problem, I was just lacking a better one

enforceLinks sounds good

Comment (BurningTreeC) [reply core/modules/widgets/link.js]: or retainLinks or keepLinks
someone other than me should decide that

**Comment by BurningTreeC** [comment]
@pmario - what I think is, that users may want to use that "linkcatcher + link to missing-tiddler" thing sometimes if they build something. This would enable that

**Pull request closed without merging**